apps/src/code-studio/pd/professional_learning_landing/LandingPage.jsx

```diff
@@ -1,15 +1,16 @@
-/**
- * Teacher Landing Page
- */
+// My Professional Learning landing page
+// studio.code.org/my-professional-learning
 
 import PropTypes from 'prop-types';
-
 import React, {Component} from 'react';
+import i18n from '@cdo/locale';
+import color from '@cdo/apps/util/color';
 import ProfessionalLearningCourseProgress from './ProfessionalLearningCourseProgress';
 import {UnconnectedTwoColumnActionBlock as TwoColumnActionBlock} from '@cdo/apps/templates/studioHomepages/TwoColumnActionBlock';
 import {EnrolledWorkshops} from './EnrolledWorkshops';
 import {pegasus} from '@cdo/apps/lib/util/urlHelpers';
-import i18n from '@cdo/locale';
+import HeaderBannerNoImage from '@cdo/apps/templates/HeaderBannerNoImage';
+import style from '../../../../style/code-studio/pl_landing_page.module.scss';
```

Comment:
nit: you can move `pl_landing_page.module.scss` into this directory then just use `import style from './pl_landing_page.module.scss'` (we've generally kept the the SCSS files next to the JS files).

Reply:
Definitely! Should I rename that scss file to fit any naming conventions? Like `LandingPage.module.scss`?

Reply:
For extra fun, it looks like we don't have a consistent style on whether we use one scss file per react file or one scss file per directory. I've done the latter, but music lab looks like [its doing the former](https://github.com/code-dot-org/code-dot-org/tree/staging/apps/src/music/views).

So, feel free to choose! I'd either rename to `professional_learning_landing.scss` to match the directory or `landingPage.module.scss` (we also don't have any conventions about casing -- even in that music lab folder linked above, there are several).

Reply:
Very interesting…I think I'll do one scss file per component for now, but if it seems like it'd make more sense to use one per directory I'll refactor.

And I like the camel case version! 😎 I'll change it to that.